### PR TITLE
feat(bug-report): ask for confirmation before opening issue

### PR DESCRIPTION
**Pull request by davidkna**
#### Description

Privacy concerns have been voiced about the privacy tool (https://github.com/starship/starship/issues/3757). To avoid any data from being exposed unintentionally, ask for confirmation before forwarding issue reports to GitHub.
Unlike https://github.com/starship/starship/issues/3794, this PR avoids complexity by simply asking for confirmation instead of allowing edits of generated reports.

#### Motivation and Context

Closes https://github.com/starship/starship/issues/3757
Closes https://github.com/starship/starship/issues/3794

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [x] I have tested using **Windows**

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: src/bug_report.rs [hunk 1]
```diff
@@ -17,16 +18,33 @@ pub fn create() {
...
+    let issue_body = get_github_issue_body(&environment);
+
+    println!(
+        "{}\n{issue_body}\n\n",
+        Style::new().bold().paint("Generated bug report:")
+    );
+    println!("Forward the pre-filled report above to GitHub in your browser?");
+    println!("{} To avoid any sensitive data from being exposed, please review the included information before proceeding. Data forwarded to GitHub is subject to GitHub's privacy policy.", Style::new().bold().paint("Warning:"));
+    println!(
+        "Enter `{}`, or anything else to decline, and `{}` to confirm your choice:\n",
```
Comment (andytom) [inline src/bug_report.rs:30]: Nitpick but maybe this should be something like 
```suggestion
        "Enter `{}` to accept, or anything else to decline, and `{}` to confirm your choice:\n",
```

File: src/bug_report.rs [hunk 2]
```diff
@@ -17,16 +18,33 @@ pub fn create() {
...
+        Style::new().bold().paint("y"),
+        Style::new().bold().paint("Enter key")
+    );
+
+    let mut input = String::new();
+    let _ = std::io::stdin().read_line(&mut input);
+
+    if input.trim() == "y" {
```
Comment (andytom) [inline src/bug_report.rs:38]: Do we want to accept if the user enters `Y`?

```suggestion
    if input.trim().to_lowercase() == "y" {
```

**Review by andytom — APPROVED** [review]
Have left some suggestions but no blockers

File: src/bug_report.rs (hunk 1 above)
Comment (davidkna) [reply src/bug_report.rs]: Thanks for the suggestions!

**Pull request merged**